setup.py

```diff
@@ -51,25 +51,10 @@ def finalize_options(self):
     cargo_args = shlex.split(os.getenv("MATURIN_SETUP_ARGS", ""))
 
 setup(
-    name="maturin",
-    author="konstin",
-    author_email="konstin@mailbox.org",
-    url="https://github.com/pyo3/maturin",
-    description="Build and publish crates with pyo3, cffi and uniffi bindings as well as rust binaries as "
-    "python packages",
     long_description=long_description,
     long_description_content_type="text/markdown",
```

Comment:
long description should also be removed now we use `readme` in `pyproject.toml`.